packages/date/src/test/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Internal dependencies
+ */
+import { isInTheFuture, getDate, setSettings, __experimentalGetSettings } from '../';
+
+describe( 'isInTheFuture', () => {
+	it( 'should return true if the date is in the future', () => {
+		const date = new Date( Number( getDate() ) + ( 1000 * 60 ) ); // 1 minute in the future
```

Comment:
These trailing comments to me are less clear, something like:

```suggestion
		// Create a Date object 1 minute in the future.
		const date = new Date( Number( getDate() ) + ( 1000 * 60 ) );
```

is a bit clearer. Also what's with the `1000` here? Shouldn't a minute be `60 * 60`? Or am I missing something?

Reply:
Ooohhhh I saw below it seems this is in milliseconds.

I think that's worth highlighting, because I too forgot that's how this worked. 😅